### ISSUE TITLE
Mention Channels [V5.11] - crashes discord when right clicking on a channel.

### ISSUE THREAD

**Issue by XxUnkn0wnxX** [Setloth/MentionChannels#16]
all tests were done on BD Branch: [fix/tooltip-crash (26161ec29fff2dc3649f40d20a3db8df1e0fd83b)](https://github.com/BetterDiscord/BetterDiscord/commit/26161ec29fff2dc3649f40d20a3db8df1e0fd83b) - Current BD Hotfix

```
Error: Error: Menu API only allows Items and groups of Items as children. Received [object Object] (object) instead
```
<img width="1091" alt="ConsoleLog" src="https://user-images.githubusercontent.com/2242880/222055591-7f2505da-c283-4af0-8476-af57f2b322a7.png">

<img width="1741" alt="Discord Crash" src="https://user-images.githubusercontent.com/2242880/222056191-3b38ad99-e416-4da0-967c-44aac588e848.png">

[debug.log](https://github.com/Sploric/MentionChannels/files/10857517/debug.log)

**Comment by Sploric** [comment]
It looks like the internal API used by my plugin has an issue currently, I will look into it.

**Comment by Sploric** [comment]
This is fixed with BetterDiscord v1.8.5 [install here](https://betterdiscord.app/)
